runatlantis.io/docs/server-configuration.md

```diff
@@ -342,6 +342,13 @@ Values are chosen in this order:
   ```
   Path to a GitHub App PEM encoded private key file. If set, GitHub authentication will be performed as [an installation](https://developer.github.com/v3/apps/installations/).
 
+- ### `--gh-app-key`
+  ```bash
+  export PRIVATE_KEY=$(cat path/to/app-key.pem)
+  atlantis server --gh-app-key=$PRIVATE_KEY
+  ```
+  The PEM encoded private key for the GitHub App.
```

Comment:
Might not be a risk in many cases, but passing secret values as plain text via flags could potentially leak these values, let's add a warning just in case! 

```suggestion
  The PEM encoded private key for the GitHub App.

  ::: warning SECURITY WARNING
  The contents of the private key will be visible by anyone that can run `ps` or look at the shell history of the machine where Atlantis is running. Use `--gh-app-key-file` to mitigate that risk.
  :::
```